infra/components/emr_serverless_analytics/__init__.py

```
@@ -0,0 +1,428 @@
+    This component provides:
+    - EMR Serverless Application (auto-start, auto-stop)
+    - S3 buckets for job artifacts and analytics output
+    - IAM roles with appropriate permissions
+    - CodeBuild project for packaging Python dependencies
+    """
+
+    def __init__(
+        self,
+        name: str,
+        *,
+        langsmith_export_bucket_name: Input[str],
```

Ruff (0.14.10)

Unused method argument: `langsmith_export_bucket_name`

(ARG002)